app/hotels/src/navigation/allHotels/SearchResultsHeader.js

```diff
@@ -0,0 +1,95 @@
+// @flow strict
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { Translation, DateFormatter } from '@kiwicom/mobile-localization';
+import {
+  StyleSheet,
+  Color,
+  Text,
+  AdaptableBadge,
+} from '@kiwicom/mobile-shared';
+
+import MapHeaderButton from './MapHeaderButton';
+import HotelsContext from '../../HotelsContext';
+
+type Props = {|
+  +goToMap: () => void,
+|};
+
+export default function SearchResultsHeader(props: Props) {
+  return (
+    <HotelsContext.Consumer>
+      {({ cityName, checkin }) => {
+        return (
+          <View style={styles.container}>
+            <View style={styles.row}>
+              <View>
+                <Text style={styles.text}>
+                  <Translation passThrough={cityName || ''} />
+                </Text>
+                {checkin !== null && (
+                  <AdaptableBadge
+                    style={styles.badge}
+                    textStyle={styles.badgeText}
+                    translation={
+                      <Translation
+                        passThrough={DateFormatter(checkin).formatCustom({
+                          weekday: 'long',
+                          day: '2-digit',
+                          month: 'long',
+                        })}
+                      />
+                    }
+                  />
+                )}
+              </View>
+              {checkin !== null && (
```

Comment:
Not showing the button, so navigation to that hotels map is not possible, then showing only error on search result screen is sufficient.